console/console-init/ui/src/modules/iot-device/components/CreateMetadata/CreateMetadata.tsx

```diff
@@ -6,18 +6,20 @@
 import React, { useState, useEffect } from "react";
 import { Button, GridItem, Grid } from "@patternfly/react-core";
 import { PlusCircleIcon } from "@patternfly/react-icons";
-import { MetaDataHeader } from "./MetaDataHeader";
 import { MetaDataRow } from "./MetaDataRow";
 import { deviceRegistrationTypeOptions } from "modules/iot-device/utils";
+import { uniqueId } from "lodash";
```

Comment:
Can we reuse the uniqueid method of utils?

Reply:
> Can we reuse the uniqueid method of utils?

yes, I think we should use utils uniqueId and use lodash uniqueId internal in utils uniqueId function. It will help in case we need to have other format then we will have to change at single places.

Reply:
I don't see much value in creating an additional unique id method, when there is something already existing in lodash. My proposal would be to use lodash unique id everywhere. Wdyt? @ajaypratap003 @rkpattnaik780